src/pages/DetailsPage.js

```diff
@@ -71,20 +74,20 @@ const DetailsPage = ({personalDetails, route, translate}) => {
                                 />
                             </View>
                             <Text style={[styles.displayName, styles.mt1, styles.mb6]} numberOfLines={1}>
-                                {details.displayName
-                                    ? details.displayName
-                                    : null}
```

Comment:
So, before when `details.displayName` was falsy, we would render `null`, now we would render `details.displayName`, is that what we want? Are there uninteded consequences of that? Mostly, I am wondering why we had this ternary condition here if there are no other consequences.

Reply:
I need to pull latest changes and I think thus this ternary would change. Why I did this as I believe we will never has a user who don't have a handle. In other words, can you imagine chating with a user who has no name(login | phone number). So if we are ended up null then we are not doing it correctly. Anyways, this issues has been fixed recently. I will pull the changes.

Reply:
I have updated it so return as null when there is no displayName.